src/openforms/registrations/contrib/zgw_apis/migrations/0013_set_zgw_api_group.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 4.2.11 on 2024-06-26 10:03
+
+from django.db import migrations
+
+from django.db.migrations.state import StateApps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def set_zgw_api_group(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    """Change the ZGW registration options to always have a ZGW API group set."""
+
+    FormRegistrationBackend = apps.get_model("forms", "FormRegistrationBackend")
+    ZgwConfig = apps.get_model("zgw_apis", "ZgwConfig")
+
+    zgw_config = ZgwConfig.objects.first()
+    if zgw_config is not None:
+        default_group = zgw_config.default_zgw_api_group
+    else:
+        default_group = None
+
+    for registration_backend in FormRegistrationBackend.objects.filter(
+        backend="zgw-create-zaak"
+    ):
+        zgw_api_group = registration_backend.options.get("zgw_api_group")
+        if zgw_api_group is None:
+            # This should not happen and is guaranteed by an upgrade check:
+            assert default_group is not None
```

Comment:
it can and will happen if a form was imported and the configs are deleted, or the config was never set up but a form from another env was imported (on our test env).

The upgrade checks can also be bypassed since they run on the system check framework, so let's be a bit more defensive here - if no default group exists but there are matching registration backends -> just create an empty group, the rest will be picked up by digest email and/or runtime crashes on the form.

The latter is preferred over not being able to `migrate`, since the problem can be fixed in the UI once the instance is updated.